config.go

```diff
@@ -108,8 +108,10 @@ type Config struct {
 	//
 	// NOTE: when not using nsqlookupd, LookupdPollInterval represents the duration of time between
 	// reconnection attempts
-	LookupdPollInterval time.Duration `opt:"lookupd_poll_interval" min:"10ms" max:"5m" default:"60s"`
-	LookupdPollJitter   float64       `opt:"lookupd_poll_jitter" min:"0" max:"1" default:"0.3"`
+	LookupdPollInterval      time.Duration `opt:"lookupd_poll_interval" min:"10ms" max:"5m" default:"60s"`
+	LookupdPollJitter        float64       `opt:"lookupd_poll_jitter" min:"0" max:"1" default:"0.3"`
+	LookupdPollAliveDuration time.Duration `opt:"lookupd_pool_alive_duration" default:"60s"`
+	LookupdPollTimeout       time.Duration `opt:"lookupd_poll_timeout" default:"6s"`
```

Comment:
we might need to make this longer by default ... apparently some clusters have thousands of topics and channels which can slow down nsqlookupd pretty badly: https://github.com/nsqio/nsq/pull/1208